src/plugins/unified_doc_viewer/public/components/doc_viewer_table/legacy/table.tsx

```diff
@@ -116,8 +116,7 @@ export const DocViewerLegacyTable = ({
       items={items}
       columns={tableColumns}
       rowProps={onSetRowProps}
-      pagination={false}
-      responsive={false}
+      responsiveBreakpoint={false}
```

Comment:
Ah thanks for flagging that, I added flex wrapping CSS to actions in the refactors to prevent overflow. You can either override that CSS or `flex-wrap: nowrap` or increase the width. If no objections I'll increase the width in this PR!

Reply:
Increasing the width sounds good to me, thanks!

Reply:
https://github.com/elastic/kibana/pull/180514/commits/8d9b3627a7609bc13fa8283984e5672a2a0d6b36

Reply:
Looking at this again I think the problem could be in the larger cell paddings than before. Also icons and text is now misaligned. 
<img width="500" alt="Screenshot 2024-04-16 at 20 10 18" src="https://github.com/elastic/kibana/assets/1415710/01a42d18-76fa-49c0-989f-7c81f56b2145">

I can work on this tomorrow as it should not go to any serverless deploy.

Reply:
Ahhh okay that's a custom cell CSS override in that case (see PR description). I've fixed it in https://github.com/elastic/kibana/pull/180514/commits/4055eff2c627bb8fee1bee32d8e21a4425b245de, thank you for flagging that!

Reply:
Perfect, thanks!